water_probe_acquanativa_ap3.orogen

```diff
@@ -12,44 +14,27 @@ import_types_from "water_probe_acquanativa_ap3Types.hpp"
 #
 # using_library "water_probe_acquanativa_ap3"
 # import_types_from "water_probe_acquanativa_ap3/CustomType.hpp"
-
-# If this project uses data types that are defined in other oroGen projects,
-# these projects should be imported there as well.
-import_types_from "std"
-# import_types_from "base"
+using_library "water_probe_acquanativa_ap3"
+import_types_from "water_probe_acquanativa_ap3/ProbeMeasurements.hpp"
 
 # Declare a new task context (i.e., a component)
 #
 # The corresponding C++ class can be edited in tasks/Task.hpp and
 # tasks/Task.cpp, and will be put in the water_probe_acquanativa_ap3 namespace.
-task_context "Task" do
+task_context "Task", subclasses: "iodrivers_base::Task" do
     # This is the default from now on, and should not be removed. Rock will
     # transition to a setup where all components use a configuration step.
     needs_configuration
 
-    # A configuration property (here, a std::string). Its value can be retrieved
-    # in the C++ code with # _config_value.get() and _config_value.set(new_value).
-    property "config_value", "/std/string"
+    # Data output, each field reflects a mesurement
+    output_port "data", "water_probe_acquanativa_ap3/ProbeMeasurements"
```

Comment:
Chose a better name for this output port

Reply:
Do you have any suggestion? I did that this way, because later (when used in a composition) it will receive the suffix `_out` and will became `data_out`. Did I get that wrong?

Reply:
No, "data" is just too general, normally we try to name the port to specify exactly what it is. Maybe "probe_measurements"